packages/devextreme/js/__internal/ui/m_validation_engine.ts

```diff
@@ -562,10 +563,10 @@ const ValidationEngine = {
     this.addGroup();
```

Comment:
[broken behavior]
This default group should be not removable. It was previously handled [here](https://github.com/DevExpress/DevExtreme/pull/27435/files#diff-64529d0ddad2c54300c7f438acc04e59f59c839d7b324b6ded1f630639c8a014L786).
Please, set isRemovable=false and write a test

Reply:
[dce6037](https://github.com/DevExpress/DevExtreme/pull/27435/commits/dce603761de8298fa7f7c8b136b4c369491a6cc4)

I believe the test is already being handled in validationEngine.test - Groups: Simple group - remove validator registration